staging/src/github.com/kubeedge/beehive/pkg/core/channel/context_channel.go

```diff
@@ -173,7 +173,6 @@ func (ctx *Context) SendToGroup(moduleType string, message model.Message) {
 		case ch <- message:
 		default:
 			klog.Warningf("The module %s message channel is full, message: %+v", module, message)
-			ch <- message
```

Comment:
If the channel is full, should we wait to write or discard? If just discard, the message maybe lost? If wait to write, whether the order of the message can be ensured between multi goroutine?

Reply:
yes, i think we can just drop the messages when the message channel is full.

Reply:
Need to check the design model or where use the sendToGroup, whether we should discard :)